docs/core/tools/cli-msbuild-architecture.md

```diff
@@ -13,7 +13,7 @@ ms.assetid: 7fff0f61-ac23-42f0-9661-72a7240a4456
 
 # High-level overview of changes in the .NET Core tools
 
-This document will describe in high-level the changes that moving from *project.json* to MSBuild and *.csproj* project system bring. It will outline the new way the tooling is layered all-up and which new pieces are available and what is their place in the overall picture. After reading this article, you should have a better understanding of all of the pieces that make up .NET Core tooling after moving to MSBuild and *.csproj*. 
```

Comment:
I chose that phrasing because @richlander described these in the blog that way: https://blogs.msdn.microsoft.com/dotnet/2017/03/07/announcing-net-core-tools-1-0/. To me, it takes all the guesswork out of what+when the change occurred to spell it all out like this; however, I'm ready to update based @mairaw and @richlander feedback.